orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java

```diff
@@ -37,13 +42,31 @@
 public class UserConfiguredUrlRestrictions {
   @Data
   public static class Builder {
-    private String allowedHostnamesRegex = ".*";
+    private String allowedHostnamesRegex =
+        ".*\\..+"; // Exclude anything without a dot, since k8s resolves single-word names
     private List<String> allowedSchemes = new ArrayList<>(Arrays.asList("http", "https"));
     private boolean rejectLocalhost = true;
     private boolean rejectLinkLocal = true;
     private HttpClientProperties httpClientProperties = new HttpClientProperties();
     private List<String> rejectedIps =
-        new ArrayList<>(); // can contain IP addresses and/or IP ranges (CIDR block)
+        // Exclude all well-known private address ranges
+        List.of(
+            "10.0.0.0/8",
+            "172.16.0.0/12",
+            "192.168.0.0/16",
+            "fc00::/7",
+            "fd00::/8"); // can contain IP addresses and/or IP ranges (CIDR block)
+
+    // Blanket exclusion on certain domains
+    // This default pattern will exclude anything that is suffixed with the excluded domain
+    private String excludedDomainTemplate = "(?=.+\\.%s$).*\\..+";
+    private List<String> excludedDomains = List.of("spinnaker", "local", "internal");
```

Comment:
I could see adding... localdomain svc.local and some other common patterns, but a good start ;)

Reply:
`svc.local` is already covered by `local`.  I'm not familiar with anything that uses `localdomain`, though - which tooling typically uses that?

Reply:
Localdomain was an oldddd unix local reference. Still around I think PRETTY regularly. 
 https://bbs.archlinux.org/viewtopic.php?id=156064 for example.... swear I've seen it even in hosts recently

Reply:
I've added `localdomain` by default - doesn't seem to me like there is any downside.